ReplayKitExample/ReplayKitExample/ViewController.swift

```diff
@@ -90,10 +85,41 @@ class ViewController: UIViewController {
         }
     }
 
+    override func viewWillAppear(_ animated: Bool) {
+        super.viewWillAppear(animated)
+        self.navigationController?.setNavigationBarHidden(false, animated: animated)
+    }
+
+    @IBAction func pickDocument(_ sender: Any) {
+        let documents = [AVFileType.mov.rawValue, AVFileType.mp4.rawValue, AVFileType.m4v.rawValue]
+        let pickerVC = UIDocumentPickerViewController(documentTypes: documents, in: .`import`)
+        pickerVC.delegate = self
+        self.navigationController?.present(pickerVC, animated: true, completion: nil)
+    }
+
+    @IBAction func browseWeb(_ sender: Any) {
+        let url = URL(string: "https://www.apple.com")!
+        let config = SFSafariViewController.Configuration()
+        config.barCollapsingEnabled = false
+        let safariVC = SFSafariViewController(url: url, configuration: config)
+        safariVC.delegate = self
+        self.navigationController?.pushViewController(safariVC, animated: true)
+        self.navigationController?.setNavigationBarHidden(true, animated: true)
+    }
+
     @available(iOS 12.0, *)
     func setupPickerView() {
         // Swap the button for an RPSystemBroadcastPickerView.
         #if !targetEnvironment(simulator)
+        // iOS 13.0 has a bug which causes a crash when RPSystemBroadcastPickerView is used to start a broadcast.
```

Comment:
Is there a link to developer forum? worth mentioning here I guess

Reply:
Added an SO link, I didn't discover the issue via dev forums but there might be another post there.